packages/origin-energy-api/package.json

```diff
@@ -0,0 +1,49 @@
+{
+    "name": "@energyweb/origin-energy-api",
+    "version": "0.0.1",
```

Comment:
Please add `private: true` for now, we would remove it when we get it fully integrated into the rest of the system.

Reply:
fixed